bin/theano-test

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env python
-import theano_test
-theano_test.main()
```

Comment:
If you remove those files, you have to remove references to it from `setup.py` as well.